lib/hammer_cli/validator.rb

```diff
@@ -98,13 +142,17 @@ def all(*to_check)
     end
 
     def option(to_check)
-      all(to_check)
+      OneOptionConstraint.new(@options, to_check)
```

Comment:
Thanks for checking it! It's breaking change but since it changes only the messages and it imho changes them for better, I'd keep it in.